src/js/tabs/motors.js

```diff
@@ -921,7 +922,7 @@ motors.initialize = async function (callback) {
 
             function disableMotorTest(e) {
                 if (motorsEnableTestModeElement.is(':checked')) {
-                    if (!ignoreKeys.includes(e.code)) {
+                    if (!ignoreKeys.includes(e.which)) {
```

Comment:
Wrote this function using `e.code` because 
https://medium.com/@danziger/both-e-which-and-e-keycode-ab9252580d08